tests/compare_INIT.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+import subprocess
+import sys
+import os.path
+
+from ert.ecl import EclFile
+from ert.test import TestAreaContext
+
+
+def compare_files( flow_file , ref_file , kw_list):
+    flow = EclFile( flow_file )
+    ref = EclFile( ref_file )
+
+    for kw in kw_list:
+        flow_kw = flow[kw][0]
+        ref_kw = ref[kw][0]
+
+        if flow_kw.equal_numeric( ref_kw , epsilon = 1e-3 ):
+            print("%s: OK" % kw)
+        else:
+            sys.exit("Keyword:%s was different in flow simulation and reference")
```

Comment:
Also: asserts are for catching programmer errors and may be left out of a "release build". Not sure how that carries over to Python, but I guess it's similar.